### PR TITLE
Initial build with CRA

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "dependencies": {
+    "node-sass": "^4.12.0",
```
Comment (squidsoup) [inline package.json]: You should probably just pin these dependencies now, otherwise renovate will be angry with you.

File: src/index.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import 'node_modules/vanilla-framework/scss/vanilla';
```
Comment (squidsoup) [inline src/index.scss]: I think this can just be `@import "vanilla"`?

File: package.json (hunk 1 above)
Comment (hatched) [reply package.json]: Done

File: src/index.scss (hunk 2 above)
Comment (hatched) [reply src/index.scss]: It complains that it's not able to find it and won't build when I do that. You might be mistaking it with `@include vanilla` to import the whole framework.

**Comment by hatched** [comment]
Thanks @squidsoup

**Pull request merged**